tenseal/cpp/context/sealcontext.cpp

```diff
@@ -28,16 +28,15 @@ EncryptionParameters create_bfv_parameters(size_t poly_modulus_degree,
 
 EncryptionParameters create_ckks_parameters(size_t poly_modulus_degree,
                                             vector<int> coeff_mod_bit_sizes) {
-    EncryptionParameters parameters(scheme_type::CKKS);
+    EncryptionParameters parameters(scheme_type::ckks);
     parameters.set_poly_modulus_degree(poly_modulus_degree);
     parameters.set_coeff_modulus(
         CoeffModulus::Create(poly_modulus_degree, coeff_mod_bit_sizes));
 
     return parameters;
 }
 
-shared_ptr<seal::SEALContext> create_context(EncryptionParameters parms) {
-    auto context = SEALContext::Create(parms);
-    return context;
+SEALContext create_context(EncryptionParameters parms) {
+    return SEALContext(parms);
```

Comment:
Is there any reason for this change?

Reply:
SEALContext::Create was removed in this release

Reply:
This is their commit
https://github.com/microsoft/SEAL/commit/a94374d348fb28390e6191cbed52830d67617736#diff-797d0099b848a1dbe9380766c6034f9f500c85b708ec931ea6dc7aa9520f3767

Reply:
Thanks! I couldn't find anything about it in the release note, as I was wondering if it's only a design choice, or that there is implications in having multiple holders of the same context.

Reply:
While is not great to change the API that much, they do not need a shared ptr, their context is pretty light.